VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/confirmation/yes-no-popup-base.component.scss

```diff
@@ -12,9 +12,19 @@
 }
 .vh-popup {
   padding: 40px;
-  margin: 250px auto;
+  @media only screen and (min-width: 768px) {
+    margin: 250px auto;
+    width: 37%;
+  }
+  @media only screen and (min-width: 280px) and (max-width: 768px) {
+    margin: 100px auto;
+    width: 70%;
+  }
+  @media only screen and (min-height: 280px) and (max-height: 768px) {
+    margin: 10px auto;
+    width: 70%;
```

Comment:
Why would the height of the screen affect the width of a component?

Reply:
this poppu is different from the others becuase the position is fixed with gray overlay in the backgroud, so we should make sure the popup in the center of the screen in horizental sreen